utils/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -106,6 +107,13 @@ class OrtAuthenticator(private val original: Authenticator? = null) : Authentica
                         }
                     }
                 }
+
+                // Then look for generic credentials passed as environment variables.
+                val usernameFromEnv = Os.env["ORT_HTTP_USERNAME"]
```

Comment:
I wonder if the name `ORT_HTTP_USERNAME` should be made more specific, e.g. it does not really indicate for what purpose these credentials are being used. How about `ORT_HTTP_PROXY_USERNAME`?

Reply:
This specifically is *not* for proxy access, which is handled by the `RequestorType.PROXY` case a few lines above. So it's really for any generic HTTP(S) download, and hence the rather generic name.